spec/components/discourse_tagging_spec.rb

```diff
@@ -184,7 +184,7 @@
 
       context 'tag synonyms' do
         fab!(:base_tag) { Fabricate(:tag, name: 'discourse') }
-        fab!(:synonym) { Fabricate(:tag, name: 'discource', target_tag: base_tag) }
```

Comment:
??

Reply:
This test case is for code that handles misspellings as the original word. Revert

Reply:
I had hoped that my cute alternative would work, but, dropping this isn't a problem. (For my tool, I'll eventually add a mask for this instance.)